### ISSUE TITLE
Node Complete Persistence

### ISSUE THREAD

**Issue by Jacki3** [LoGaCulture/LUTE#51]
**Is your feature request related to a problem? Please describe.**
Typically when engaging with location nodes, in one continous session the player will: visit a node, complete it then visit another. They may unlock items or achievements upon reaching this node and completing the challenge at that node.

The node state is saved as complete and will not execute again unless it is allowed to. However, when the player exits and returns to the game this completition state is not saved and players can return to the same location and repeat the actions (this may cause the game to break or disrupt the flow of a given experience).

**Describe the solution you'd like**
When a node is completed, if the designer chooses, the completition state should be saved and serialised so that when reloading the completition and execution state for that node is retained.

**Additional context**
Making use of the exisiting saving system, we can encode this data to a new node save class which will ensure that the related properties (and potential future ones) are saveable upon request.

**Comment by Jacki3** [comment]
Update Node.cs to include a 'Saveable' property which will ensure that the execution state and complete status (boolean) will be saved to a persistent file that will be loaded when using the pre defined save system.

Rather than having all nodes save, this option requires designers to switch it on as certain nodes always require execution on entering play (such as menus and starting mechanics).

The BasicGame prefab also uses an array of lat lons for the map now which points directly at Avebury rather than Helsinki (smoother testing).

This issue should be fixed but the wiki may want updating to reflect these changes.
